db2/README.md

```diff
@@ -1,10 +1,15 @@
-# TDK DB2 Demo
+# TDK DB2 LUW Demo
 
-Spin up databses:
+Spin up databases:
 ```shell
 docker-compose up input_db output_db -d
 ```
 
+Set up the TDK license key:
+```shell
+export SYNTHESIZED_KEY=kDpeQB...
+```
+
```

Comment:
I suggest to add an information about the license key. Maybe to add it to the Oracle demo as well:

Set up the TDK license key:
```shell
export SYNTHESIZED_KEY=kDpeQB...
```
> **Note**
>
> [Please contact us to obtain an enterprise license key.](https://www.synthesized.io/contact-sales)